djs-bot/commands/music/playlists.js

```diff
@@ -196,72 +349,252 @@ module.exports = new SlashCommand()
 				where: {
 					name: playlistName,
-					userId: interaction.user.id
-				}
+					userId: interaction.user.id,
+				},
 			});
 
-			if (!playlist) return interaction.reply({ embeds: [new EmbedBuilder().setColor(client.config.embedColor).setDescription("I couldn't find a playlist with that name")], ephemeral: true });
-			if (playlist.userId !== interaction.user.id) return interaction.reply({ embeds: [new EmbedBuilder().setColor(client.config.embedColor).setDescription("You can't remove songs from a playlist that isn't yours")], ephemeral: true });
+			if (!playlist)
+				return interaction.reply({
+					embeds: [
+						new EmbedBuilder()
+							.setColor(client.config.embedColor)
+							.setDescription(
+								"I couldn't find a playlist with that name"
+							),
+					],
+					ephemeral: true,
+				});
+			if (playlist.userId !== interaction.user.id)
+				return interaction.reply({
+					embeds: [
+						new EmbedBuilder()
+							.setColor(client.config.embedColor)
+							.setDescription(
+								"You can't remove songs from a playlist that isn't yours"
+							),
+					],
+					ephemeral: true,
+				});
 
 			const song = options.getString("song", true);
-			if (!song) return interaction.reply({ embeds: [new EmbedBuilder().setColor(client.config.embedColor).setDescription("You need to provide a song to remove")], ephemeral: true });
+			if (!song)
+				return interaction.reply({
+					embeds: [
+						new EmbedBuilder()
+							.setColor(client.config.embedColor)
+							.setDescription(
+								"You need to provide a song to remove"
+							),
+					],
+					ephemeral: true,
+				});
 			const songData = await client.db.song.findFirst({
 				where: {
-					name: song
-				}
+					name: song,
+				},
 			});
-			if (!songData) return interaction.reply({ embeds: [new EmbedBuilder().setColor(client.config.embedColor).setDescription("I couldn't find a song with that name")], ephemeral: true });
+			if (!songData)
+				return interaction.reply({
+					embeds: [
+						new EmbedBuilder()
+							.setColor(client.config.embedColor)
+							.setDescription(
+								"I couldn't find a song with that name"
+							),
+					],
+					ephemeral: true,
+				});
 
 			await client.db.playlist.update({
 				where: {
-					id: playlist.id
+					id: playlist.id,
 				},
 				data: {
 					songs: {
 						disconnect: {
-							id: songData.id
-						}
-					}
-				}
+							id: songData.id,
+						},
+					},
+				},
 			});
 
-			return interaction.reply({ embeds: [new EmbedBuilder().setColor(client.config.embedColor).setDescription(`Removed **${songData.name}** from **${playlist.name}**`)], ephemeral: true });
-		}
+			return interaction.reply({
+				embeds: [
+					new EmbedBuilder()
+						.setColor(client.config.embedColor)
+						.setDescription(
+							`Removed **${songData.name}** from **${playlist.name}**`
+						),
+				],
+				ephemeral: true,
+			});
+		} else if (operation === "play") {
```

Comment:
Refactor this horrendous if else and copy pasted abomination to use DRY principle. Anyone can continue or me later on lunch break.